hudi-client/hudi-spark-client/src/main/java/org/apache/hudi/table/action/commit/SparkInsertOverwriteCommitActionExecutor.java

```diff
@@ -66,7 +67,9 @@ public HoodieWriteMetadata<HoodieData<WriteStatus>> execute() {
   @Override
   protected Partitioner getPartitioner(WorkloadProfile profile) {
     return table.getStorageLayout().layoutPartitionerClass()
-        .map(c -> getLayoutPartitioner(profile, c))
+        .map(c -> c.equals(HoodieLayoutConfig.SIMPLE_BUCKET_LAYOUT_PARTITIONER_CLASS_NAME)
```

Comment:
No, consistentBucketIndex works correctly, it would generate different file ids.

Reply:
@beyond1920 I read `consistentBucketIndex` implementation, found it must tag incomming records to allocation fgId, so #8073 will cause some quesion

Reply:
> No, consistentBucketIndex works correctly, it would generate different file ids.

`consistentBucketIndex` can not work correctly, change the ut case

Reply:
emm, sorry for the hurry response before.
Thank you for point it out. 
I need to spend more time to get familiar with `ConsistentBucketIndex`. I would response ASAP.